.swiftlint.yml

```diff
@@ -43,7 +43,6 @@ disabled_rules: # rule identifiers to exclude from running
   - orphaned_doc_comment
   - private_unit_test
   - reduce_boolean
-  - redundant_set_access_control
```

Comment:
why did we decide to remove this?

Reply:
hmm. was not supposed to

Reply:
@cyndichin this removes it from the disabled ruleset, so that it will run https://realm.github.io/SwiftLint/redundant_set_access_control.html

Reply:
sounds good! ah sorry missed that conversation on that PR. do we want to vote as a team on disabling or enabling rules in the future?